yi-core/src/Yi/Types.hs

```diff
@@ -434,8 +434,11 @@ data Config = Config {startFrontEnd :: UIBoot,
                       bufferUpdateHandler :: !(S.Seq (S.Seq Update -> BufferM ())),
                       layoutManagers :: ![AnyLayoutManager],
                       -- ^ List of layout managers for 'cycleLayoutManagersNext'
-                      configVars :: !ConfigState.DynamicState
+                      configVars :: !ConfigState.DynamicState,
                       -- ^ Custom configuration, containing the 'YiConfigVariable's. Configure with 'configVariableA'.
+                      configDisableSystemClipboard :: Bool
```

Comment:
Make strict.

Reply:
Use strict. Thank you for your reviewing.